lib/models/user-whitelist.js

```diff
@@ -13,4 +14,28 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment().utc();
```

Comment:
NOTE: moment().isBefore() has undefined behavior and should not be used! If the code runs fast the initial created moment would be the same as the one created in isBefore to perform the check, so the result would be false. But if the code runs slower it's possible that the moment created in isBefore is measurably after the one created in moment(), so the call would return true.